app/plugins/adp-backend/src/service/deliveryProgrammeAdminRouter.test.ts

```diff
@@ -162,7 +166,7 @@ describe('createRouter', () => {
       expect(response.status).toEqual(201);
       expect(mockCatalogClient.getEntities).toHaveBeenCalledWith(
         expect.any(Object),
-        { token: 'token' },
+        expect.any(Object),
```

Comment:
Is this intentional? Why would the token not be getting passed to the catalog anymore?